src/molecule/cookiecutter.py

```diff
@@ -0,0 +1,61 @@
+"""cookiecutter main function."""
```

Comment:
I'm not a lawyer, but if this contains code that was taken from cookiecutter, doesn't it need to carry the [license](https://github.com/cookiecutter/cookiecutter/blob/master/LICENSE) and copyright from cookiecutter? (Same for other code taken from that project.)

Reply:
@felixfontein 

Thanks for your reminder.

The cookiecutter [license](https://github.com/cookiecutter/cookiecutter/blob/master/LICENSE) and copyright are added in [new commit](https://github.com/ansible-community/molecule/pull/3526/commits/fd16041eb4c9b82e8c20fe77b694abf134f0dd87), referring to how `interpolation.py` did.
https://github.com/ansible-community/molecule/blob/04157c3684a46987ba00a6223261e3da30ff5b0c/src/molecule/interpolation.py#L1-L16